packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -161,6 +161,9 @@ const runAPI = (plugin, api, args) => {
         },
       }
     }
+    let localReporter = activity
+      ? { ...reporter, panicOnBuild: activity.panicOnBuild.bind(activity) }
```

Comment:
We should bind other methods too (in a follow up of course)

Reply:
I wasn't quite sure if we needed to bind `panicOnBuild`, this should be only needed if `this` is used there (which seems like we don't use in `panicOnBuild`, but we do use in other methods - so worth doing check if we didn't break things)